rules/typescript.js

```diff
@@ -166,6 +166,15 @@ module.exports = {
         format: null,
         selector: 'variable',
       },
+      // Allow certain keywords that are used by engineering at Dott.
+      {
+        filter: {
+          match: true,
+          regex: '_version',
+        },
+        format: null,
+        selector: 'default',
+      },
```

Comment:
This should be accompanied by its own test.

In general terms, you need to add a passing fixture and a failing fixture.